src/openapi/CommonSchemas.yaml

```diff
@@ -6,18 +6,18 @@ info:
 
 components:
   schemas:
-    Password:
+    Username:
       type: string
+      maxLength: 256
+      format: email
+      description: Username/email of the account.
+    Password:
+      type: byte
```

Comment:
Hier hat sich nochmal ein Fehler eingeschlichen. 

**type: string
format: byte**

wäre die korrekte Schreibweise für Base64-Kodierte felder.

Reply:
Ja das ist richtig, aber wäre hier 'password' nicht doch besser?
String Formats -> 
password – a hint to UIs to mask the input
byte – base64-encoded characters, for example, U3dhZ2dlciByb2Nrcw==

Reply:
Ich meine auf Passwort gibt ja der Feldname schon genug Informationen. Aber für das saubere Implementieren sollte hier eher byte genommen werden. Auch für den Generator kann das ggf. Einfluss haben.